src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -253,11 +256,13 @@ export default class ComboBox extends React.PureComponent<Props, State> {
       options.length === 0 &&
       emptyState && <div className={styles.EmptyState}>{emptyState}</div>;
 
+    const active = !disabled && this.state.popoverActive;
+
     return (
       <div
         onClick={this.handleClick}
         role="combobox"
-        aria-expanded={this.state.popoverActive}
+        aria-expanded={active}
         aria-owns={this.state.comboBoxId}
         aria-controls={this.state.comboBoxId}
         aria-haspopup
```

Comment:
I'm wondering if someone from Polaris might know if there are some aria attributes that should be added to inform the user that the combobox is disabled?

Reply:
Yeah I was wondering as well... perhaps an `aria-disabled`?

Reply:
I would think that yes we would need an aria-disabled on the div since it's the one handling the interaction. cc @dpersing

Reply:
@dpersing Should we be doing both though? Disabling the text input and putting the `aria-disabled` on the div since the div is what the user clicks to open/close the combobox.

Reply:
@jfangrad It technically wouldn't hurt, but since the use case for assistive tech that could process the `aria-disabled` attribute on the `<div>` is pretty much limited to screen readers at the moment, the majority of users who would get that info would be users accessing the widget with the keyboard instead of the mouse. The `disabled` state on the input would prevent tab focus to start with, and would visually signal to sighted users that the control isn't available. I agree with @dleroux that confining disabling to just the control that triggers the interaction, and using the HTML native option, should lead to less inconsistency.